webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -209,20 +218,27 @@ class AppConfigAuth extends ApplicationContextAware {
                     }
                     value
                 }
-              } catch { case ex: ConfigException.Missing => defaultEnableRestToken }
+              } catch {
+                case _: ConfigException.Missing =>
+                  if (defaultValue != restTokenGlobalFeatureSwitch) {
+                    logger.info(
+                      s"User access to REST API via token for provider ${x.name} is by default configured to be ${defaultValue.name}, overriding the global configuration"
+                    )
+                  }
```

Comment:
without this the user will not know that the Rudder default value is `disabled` for OIDC